php/libraries/User.class.inc

```diff
@@ -532,5 +532,32 @@ class User extends UserPermissions
             (string) $this->userInfo['Password_hash']
         );
     }
+
```

Comment:
Looks good pending Travis. I have a suspicion that one of the static analysis tools won't like the two new lines here but maybe it doesn't matter.